lapis2/src/main/antlr/org/genspectrum/lapis/model/variantqueryparser/VariantQuery.g4

```diff
@@ -60,11 +60,11 @@ aaInsertionQuery: insertionKeyword gene ':' position ':' aaInsertionSymbol+;
 aaInsertionSymbol: possiblyAmbiguousAaSymbol | '?';
 
 nextcladePangolineageQuery: nextcladePangoLineagePrefix pangolineageQuery;
-nextcladePangoLineagePrefix: 'nextcladePangoLineage:' | 'NEXTCLADEPANGOLINEAGE:';
+nextcladePangoLineagePrefix: 'nextcladePangoLineage:' | 'NEXTCLADEPANGOLINEAGE:' | 'nextcladepangolineage:';
 
 nextstrainCladeLineageQuery: nextstrainCladePrefix nextstrainCladeQuery;
-nextstrainCladePrefix: 'nextstrainClade:'| 'NEXTSTRAINCLADE:';
-nextstrainCladeQuery: NUMBER NUMBER nextstrainCladeCharacter | 'RECOMBINANT';
+nextstrainCladePrefix: 'nextstrainClade:'| 'NEXTSTRAINCLADE:' | 'nextstrainclade:';
+nextstrainCladeQuery: NUMBER NUMBER nextstrainCladeCharacter | 'RECOMBINANT' | 'recombinant';
```

Comment:
I suspect that the lowercase "recombinant" version might not work yet, did you test it?